examples/existing_container_demo.py

```
@@ -0,0 +1,324 @@
+        logger.info("📦 Installing packages...")
+        session.install(["numpy", "pandas", "matplotlib"])
+        
+        # Create some files
+        logger.info("📁 Setting up files...")
+        session.execute_command("echo 'print(\"Hello from existing container!\")' > /sandbox/hello.py")
+        session.execute_command("echo 'Custom environment data' > /sandbox/data.txt")
+        
+        # Get container ID before closing
+        container_id = session.container.id
+        logger.info(f"✅ Container created with ID: {container_id[:12]}...")
+        
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)